api/python/provisioner/commands/set_swupdate_repo.py

```diff
@@ -0,0 +1,343 @@
+#
```

Comment:
I would suggest segregating all Upgrade API under a dir.
Same for command would help with code structure.

`api/python/provisioner/upgrade/set_swupdate_repo.py`
`api/python/provisioner/commands/upgrade/swupgrade.py`

Command should be a high level call and internally it should import the essential API modules.

Reply:
@dmitrydb please create a separate task. We might need to work on that quite soon.

Reply:
Separate ticket for that
https://jts.seagate.com/browse/EOS-16941